openpgp/packet/private_key.go

```diff
@@ -155,6 +156,60 @@ func mod64kHash(d []byte) uint16 {
 	return h
 }
 
+// This is the counterpart to the Decrypt() method below.
```

Comment:
`private_key.go:159:1: comment on exported method PrivateKey.Encrypt should be of the form "Encrypt ..."`

one from `golint`. Generally the comment should be phrased like: `// Encrypt is the counterpart to the Decrypt`, that's the convention.

Also please run `go fmt private_key.go`, it will reformat the code and save it automatically. It fixes one line in this file, rest are fine IIRC.